susie/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.5.0
```

Comment:
change this to 1.8.2 which more up to date computers will only have

Reply:
done!